rest/resource-server/src/main/resources/application.yml

```diff
@@ -11,6 +11,11 @@ server:
   port: 8091
 
 spring:
+  http:
+    multipart:
+      max-file-size: 500MB
```

Comment:
Maybe resize the limitations to 100MB?
Do we have any requirements for such huge file sizes?

Reply:
I think there might be source-DVDs which are easily above 500MB. I have no strong opinion whether 100MB or 500MB is the better choice.

Reply:
maybe we would touch 100MB from time to time, so 500MB is good from a use case point of view. But that requir also to have 500MB free mem or maybe 1GB free ram for such upload?

Reply:
Uploads are handled in a stream like manner, they are never loaded completly into the RAM.